quantumcat/gates/custom_gates/braket/__init__.py

```diff
@@ -0,0 +1,35 @@
+# (C) Copyright Artificial Brain 2021.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from quantumcat.gates.custom_gates.braket.u_gate import UGate
+from quantumcat.gates.custom_gates.braket.u2_gate import U2Gate
+from quantumcat.gates.custom_gates.braket.u3_gate import U3Gate
+# from quantumcat.gates.custom_gates.braket.u1_gate import U1Gate
```

Comment:
comment to be removed?

Reply:
It will give no error since line 21 has the same import. This is commented because in my system u1_gate.py is not getting recognized and gives and error.